locale/el_GR.json

```
@@ -0,0 +1,181 @@
+    "postpone-positive-integer": "Πληκτρολογήστε έναν θετικό ακέραιο αριθμό.",
+    "postpone": "Αναβολή",
+    "postpone-result-text": "%{count} κάρτες αναβλήθηκαν σε %{seconds} δευτερόλεπτα.",
+    "postpone-retention-change": "<br>Μέσος στόχος ανάκλησης των αναβληθέντων καρτών: %{prev_retention} -> %{new_retention}",
+
+    "remedy-start-date": "Επιλέξτε την ημερομηνία έναρξης",
+    "remedy-end-date": "Επιλέξτε την ημερομηνία λήξης",
+    "remedy-title": "Επιδιόρθωση κατάχρησης του κουμπιού Hard",
+    "remedy-no-hard-reviews": "Δεν υπάρχουν επαναλήψεις με βαθμολογία Hard στο επιλεγμένο εύρος ημερομηνιών.",
+    "remedy-confirmation": "%{count} επαναλήψεις είχαν βαθμολογία Hard μεταξύ %{start_date} και %{end_date}.\nΑυτές οι βαθμολογίες θα αντικατασταθούν με Again.\nΤα ID αυτών των revlogs θα αποθηκευτούν σε ένα αρχείο CSV στο φάκελο του πρόσθετου για να είναι δυνατή η αναίρεση των αλλαγών.\nΘέλετε να προχωρήσετε;",
+    "remedy-success": "%{count} επαναλήψεις επιδιορθώθηκαν.\nΠαρακαλώ βελτιστοποιήσετε εκ νέου τις παραμέτρους του FSRS για να ενσωματώσετε τις αλλαγές.",
+    "undo-remedy-no-file": "Δεν βρέθηκαν επιδιορθωμένες επαναλήψεις",
+    "undo-remedy-restored": "%{count} επαναλήψεις αποκαταστάθηκαν",
+    "undo-remedy-success": "%%{count} επαναλήψεις αποκαταστάθηκαν.\nΠαρακαλούμε να βελτιστοποιήσετε εκ νέου τις παραμέτρους του FSRS για να ενσωματώσετε τις αλλαγές.",
+
+    "x-%": "%{count}%",
+    "x-cards": {
+      "one": "1 κάρτα",
+      "other": "%{count} κάρτες"
+    },
+    "x-cards-per-hour": {
+      "one":"1 κάρτα/ώρα",
+      "other":"%{count} κάρτες/ώρα"
+    },
+    "x-days": {
+      "one": "1 ημέρα",
+      "other": "%{count} ημέρες"
+    },
+    "fsrs-status": "FSRS: %{status}",
+
+    "disperse-label": "Dispersing Siblings",
+    "disperse-progress": "%{count}/%{total} notes dispersed",
+    "disperse-cards-in": "cards in",
+    "disperse-notes": "notes dispersed",
+    "disperse-card-message": "Dispersed card %{card_id} from %{old_due} to %{new_due}",
+    "disperse-card-range": "Card %{card_id} due range: %{start_due} - %{end_due}",
+    "disperse-too-close": "Due dates are too close to disperse:",
+    "disperse-result": "%{result} in %{count} seconds",
+
+    "clear-custom-data-confirmation": "Διαγραφή προσαρμοσμένων δεδομένων σε όλες τις κάρτες; \n Ο προσαρμοσμένος προγραμματισμός του FSRS4Anki αποθηκεύει την κατάσταση μνήμης στα προσαρμοσμένα δεδομένα.\n Δεν χρησιμοποιείται όταν ενεργοποιείτε το ενσωματωμένο FSRS.\n Είστε σίγουροι;",
+    "clear-custom-data-result": "%{count} κάρτες εκκαθαρίστηκαν σε %{seconds} δευτερόλεπτα.",
+    "clear-manual-rescheduling-confirmation": "Αυτή η επιλογή αφαιρεί τις εγγραφές του revlog που δημιουργούνται από τις επιλογές \«Ορισμός ημερομηνίας λήξης\» και \«Επαναπρογραμματισμός καρτών κατά την αλλαγή\», οι οποίες προηγούνται αμέσως μιας άλλης χειροκίνητης εγγραφής ή εγγραφής επαναπρογραμματισμού.\n\nΗ ενέργεια αυτή δεν μπορεί να ανακληθεί. Θέλετε να συνεχίσετε;",
```

```suggestion
    "clear-manual-rescheduling-confirmation": "Αυτή η επιλογή αφαιρεί τις εγγραφές του revlog που δημιουργούνται από τις επιλογές «Ορισμός ημερομηνίας λήξης» και «Επαναπρογραμματισμός καρτών κατά την αλλαγή», οι οποίες προηγούνται αμέσως μιας άλλης χειροκίνητης εγγραφής ή εγγραφής επαναπρογραμματισμού.\n\nΗ ενέργεια αυτή δεν μπορεί να ανακληθεί. Θέλετε να συνεχίσετε;",
```

```
json.decoder.JSONDecodeError: Invalid \escape: line 161 column 131 (char 12152)
```

having escape characters on this line causes the addon to crash.